services/Skyra.Database/Migrations/20210422211300_V03_NewSocialFeaturesAndValueRename.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Skyra.Database.Migrations
+{
+    public partial class V03_NewSocialFeaturesAndValueRename : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "social.achieve",
```

Comment:
OOF. Any way to NOT drop this column? 😬

Reply:
What do you mean?

Reply:
He means that the data that is there right now is not being backed up but rather just completely pruned. This is now how we normally write migrations. Did you generate the migration _after_ putting some data in the database?